src/components/common/PageSearch/usePageSearch.ts

```diff
@@ -0,0 +1,53 @@
+import { ChangeEvent, useEffect } from 'react'
+
+import { formatETATime } from '../../../utils/utils'
+import { useSharedStore } from '../../SharedStoreProvider'
+
+export const usePageSearch = () => {
```

Comment:
why not moving these functions into `documentSearchStore` instead of custom hook?

Reply:
good point, moved to `documentSearchStore`